File: apps/discord-bot/src/mongodb/collections/MongoUser.cs
using System.Runtime.Serialization;
using MongoDB.Bson.Serialization.Attributes;

public class MongoUser
{
    [BsonId]
    [DataMember]
    public MongoDB.Bson.ObjectId _id { get; set; }
    /// <summary>
    /// The ID of the discord member.
    /// </summary>
    public ulong DiscordId;
    /// <summary>
    /// The username of the database user.
    /// </summary>
    public string Username;
    /// <summary>
    /// The password of the database user, given by me.
    /// </summary>
    public string Password;

    public MongoUser(ulong _DiscordId, string _Username, string _Password)
    {
        DiscordId = _DiscordId;
        Username = _Username;
        Password = _Password;
    }
}

File: apps/discord-bot/app/src/main/java/discordbot/mongodb/collections/MongoUser.java
package discordbot.mongodb.collections;

import org.bson.codecs.pojo.annotations.BsonId;
import org.bson.types.ObjectId;

import kotlin.ULong;

public class MongoUser {
    @BsonId
    public ObjectId _id;
    /**
     * The ID of the discord member.
     */
    public ULong DiscordID;
    /**
     * The username of the database user.
     */
    public String Username;
    /**
     * The password of the database user, given by me.
     */
    public String Password;

    public MongoUser(ULong _DiscordID, String _Username, String _Password) {
        DiscordID = _DiscordID;
        Username = _Username;
        Password = _Password;
    }
}